rootfs/etc/profile.d/tsh-login.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
 function tsh-login() {
-	tsh login --bind-addr=:${GEODESIC_PORT} --proxy=${TELEPORT_PROXY_DOMAIN_NAME:-tele.${DOCKER_IMAGE#*.}} $*
+	if (($# == 0)); then
+		tsh login --bind-addr=:${GEODESIC_PORT} --proxy=${TELEPORT_LOGIN_PROXY:-tele.${DOCKER_IMAGE#*.}} $STAGE
```

Comment:
Instead, we shouold export the envs like `TELEPORT_LOGIN_BIND_ADDR` so we can call `tsh login` without arguments

Reply:
https://github.com/gravitational/teleport/issues/2620#issuecomment-481756625

Reply:
And use https://github.com/gravitational/teleport/blob/7e636b0b2b14bee3b4a0103cedcf8ddde9dc082a/tool/tsh/tsh.go#L188 instead of `--proxy`